src/aioftp/__main__.py

```diff
@@ -65,6 +66,7 @@
         format="%(asctime)s [%(name)s] %(message)s",
         datefmt="[%H:%M:%S]:",
     )
+path_io_factory: Union[Type[aioftp.PathIO], Type[aioftp.MemoryPathIO]]
```

Comment:
No, mypy then reports the error "Incompatible types in assignment (expression has type "type[MemoryPathIO]", variable has type "AbstractPathIO")". This is because AbstractPathIO is abstract and it is not allowed to create objects of abstract classes.

Reply:
I don't understand. Now I don't have such an annotation causing errors. Although there were problems a few times before. I will change it to AbstractPathIO

Reply:
Fixed